ctapipe/image/extractor.py

```diff
@@ -199,7 +200,8 @@ def extract_sliding_window(waveforms, width, sampling_rate_ghz, sum_, peak_time)
     peak_time[0] /= sampling_rate_ghz
 
 
-@njit(cache=True)
+# FIXME JIT has been disabled because it asserts when passing floating-point waveforms (e.g., FlashCamExtractor, at least on arm64)
```

Comment:
It asserts what?

Reply:
Nothing anymore - was probably due to a shape mismatch in an earlier version.